lighthouse-extension/url-transform.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const through = require('through2');
+
+/**
+ * This is a browserify transform that looks for requires to url and whatwg-url
+ * and replaces them for the browser version in Lighthouse. This is just for the extension
```

Comment:
could you instead try a global first?

`const URL = self.URL || require('url').URL || require('whatwg-url').URL;`  

from my testing, it looks safe and solid. you can nuke this transform as long as the require looks like the above.
and you should put `whatwg-url` in the browserify ignores.